src/app/month/page.tsx

```diff
@@ -0,0 +1,36 @@
+async function getData() {
+  const res = await fetch('http://localhost:3000/api/months?userId=64623b1099fb59513caf45c0');
```

Comment:
You can create a service and call just hooks instead of doing a http request in each page

Reply:
Have a look to the updated structure and let me know if you still think i should use react query